src/lib/rehype-code-plugins.ts

```diff
@@ -41,6 +43,16 @@ export const rehypeCodePlugins: Pluggable[] = [
 							this.addClassToHast(node, 'empty-line')
 						}
 					},
+					span(node, _line: number, col: number) {
+						// prevent user-select on the start +/- in diff lines
+						if (this.options.lang === 'diff' && col === 0) {
+							const firstChild = node.children[0] as any
```

Comment:
Not happy with this, but TS complained when I casted to Element. 😢

Reply:
👎 at TS for this! I was just about to suggest Element 😅